app/web/src/pages/HomePage/HomePage.tsx

```diff
@@ -9,6 +9,7 @@ const ProjectsPage = () => {
         title="Home page"
         description="Learn about Code CAD and the CadHub community"
         lang="en-US"
+        socialImageUrl="https://cadhub.xyz/default-social-image.jpg"
```

Comment:
for now its:
![default-social-image](https://user-images.githubusercontent.com/29681384/132972540-49c0e764-825d-4ef2-b901-b0b9d4e04daf.jpg)